igor_src/main.cpp

```diff
@@ -45,9 +45,73 @@
 #include <chrono>
 #include <set>
 
+#include <string>
+//#include "CDR3SeqData.h"
 
 using namespace std;
 
+
+// FIXME: PUT THIS IN ANOTHER FILE
```

Comment:
All this indeed shouldn't lie in main.cpp
The main file should ideally be 100% dedicated to IGoR's execution. Although already a bit messy the Utils.h/cpp files would be a much better place for such utility definitions.